src/conn.rs

```diff
@@ -232,15 +237,15 @@ impl<const N: usize, P: ConnectionPeer> Connection<N, P> {
         mut writes: mpsc::UnboundedReceiver<Write>,
         mut shutdown: oneshot::Receiver<()>,
     ) -> io::Result<()> {
-        tracing::debug!("uTP conn starting... {:?}", self.cid.peer);
+        tracing::debug!("uTP conn starting... {:?}", self.cid.peer_id);
```

Comment:
Is it still valuable to see the `ConnectionPeer` embedded in the `Peer` here? I imagine it could be, sometimes
```suggestion
        tracing::debug!("uTP conn starting... {:?}", self.peer);
```